bitnami/spark/templates/metrics-config-map.yaml

```diff
@@ -0,0 +1,19 @@
+{{- if .Values.metrics.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: metrics-config
+  labels:
+    app.kubernetes.io/name: {{ include "spark.name" . }}
+    helm.sh/chart: {{ include "spark.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name | quote }}
+    app.kubernetes.io/managed-by: {{ .Release.Service | quote }}
+data:
+  metrics.properties: |-
+    *.source.jvm.class=org.apache.spark.metrics.source.JvmSource
+    *.sink.prometheusServlet.class=org.apache.spark.metrics.sink.PrometheusServlet
+    *.sink.prometheusServlet.path=/metrics/prometheus
+    master.sink.prometheusServlet.path=/metrics/master/prometheus
+    applications.sink.prometheusServlet.path=/metrics/applications/prometheus
+
+{{- end -}}
```

Comment:
Thanks for adding this. I think we should also add a property to specify a custom config map for the metrics, similar to the `configurationConfigMap` in the master and the workers, to have an easy way to modify or override the default configuration.

It could be `metrics.metricsConfigMap`. Could be set like this in the `values.yaml` (don't forget to add it to the README too and to change the logic when mounting the config map): 

```
metrics:
  enabled: false
  ## Set a custom metrics.properties by using an existing configMap with the configuration file.
  ##
  # metricsConfigMap:
...
```

And then this default config map would only be created `{{- if and .Values.metrics.enabled (not .Values.metrics.metricsConfigMap) }}`

Reply:
fixed